src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -482,13 +485,21 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
     }
 
     public render() {
-        return (
-            <PageLayout noMargin={true}>
-                {
-                    this.pageContent
-                }
-            </PageLayout>
-        )
+
+        if (this.resultsViewPageStore.studies.isComplete && !this.resultsViewPageStore.tabId) {
+            setTimeout(()=>{
```

Comment:
i would be nicer to handle this redirect in router but we need resultsViewStore data to determine where to send user.

Reply:
theres no need to debounce/clear this bc its idempotent right?

Reply:
Yes, if the tabId route was set multiple times, it wouldn't have any affect.  
It's also highly unlikely (maybe impossible) that render would be called twice in same frame.  And the code run in the timeout sets tabId (next frame).  So only change this could run twice is if another render was able to execute in same frame.